framework/src/application/node/node.ts

```diff
@@ -50,16 +52,135 @@ import { EventPostTransactionData } from '../../types';
 import { InMemoryChannel } from '../../controller/channels';
 import { EventInfoObject } from '../../controller/event';
 import { ApplicationState } from '../application_state';
+import { accountAssetSchema, defaultAccountAsset } from './account';
+import {
+	EVENT_PROCESSOR_BRADCASRT_BLOCK,
+	EVENT_PROCESSOR_SYNC_REQUIRED,
+} from './processor/processor';
+import { EVENT_SYNCHRONIZER_SYNC_RQUIRED } from './synchronizer/base_synchronizer';
 
 const forgeInterval = 1000;
 const { EVENT_NEW_BLOCK, EVENT_DELETE_BLOCK } = chainEvents;
 const { EVENT_ROUND_CHANGED } = dposConstants;
 const { EVENT_TRANSACTION_REMOVED } = txPoolEvents;
 
-export interface GenesisBlockInstance extends GenesisBlockJSON {
+interface Payload {
+	id: string;
+	senderPublicKey: string;
+	nonce: string;
+	fee: string;
+	signatures: string[];
+}
+
+interface TransferPayload extends Payload {
+	type: 8;
+	asset: {
+		recipientAddress: string;
+		amount: string;
+	};
+}
+
+interface DelegatePayload extends Payload {
+	type: 10;
+	asset: {
+		username: string;
+	};
+}
+
+interface VotePayload extends Payload {
+	type: 13;
+	asset: {
+		votes: {
+			delegateAddress: string;
+			amount: string;
+		}[];
+	};
+}
+
+export interface GenesisBlockJSON {
 	readonly communityIdentifier: string;
+	readonly header: {
+		readonly id: string;
+		readonly version: number;
+		readonly timestamp: number;
+		readonly height: number;
+		readonly previousBlockID: string;
+		readonly transactionRoot: string;
+		readonly generatorPublicKey: string;
+		readonly reward: string;
+		readonly asset: {
+			seedReveal: string;
+			maxHeightPreviouslyForged: number;
+			maxHeightPrevoted: number;
+		};
+		readonly signature: string;
+	};
+	readonly payload: Array<TransferPayload | DelegatePayload | VotePayload>;
 }
 
+const convertGenesisBlock = (genesis: GenesisBlockJSON): Block => {
```

Comment:
maybe `genesisBlockToJSON`? or `blockToJSON`?

Reply:
Same as above, this is temporal solution until #5234